scripts/documentation/plugin-quality-indicators.ts

```diff
@@ -75,45 +82,38 @@ main();
 // Publish date (and github directory, readme content)
 // i.e. https://registry.npmjs.org/@nxkit/playwright
 async function getNpmData(plugin: PluginRegistry, skipNxVersion = false) {
-  try {
-    const { data } = await axios.get(
-      `https://registry.npmjs.org/${plugin.name}`
-    );
-    const lastPublishedDate = data.time[data['dist-tags'].latest];
-    const nxVersion = skipNxVersion || (await getNxVersion(data));
-    if (!data.repository) {
-      console.warn('- No repository defined in package.json!');
-      return { lastPublishedDate, nxVersion, githubRepo: '' };
-    }
-    const url: String = data.repository.url;
-    const githubRepo = url
-      .slice(url.indexOf('github.com/') + 11)
-      .replace('.git', '');
-    return {
-      lastPublishedDate,
-      githubRepo,
-      nxVersion,
-      // readmeContent: plugin.name
-    };
-  } catch (ex) {
-    return { lastPublishedDate: '', githubRepo: '' };
+  const { data } = await axios.get(`https://registry.npmjs.org/${plugin.name}`);
+  const lastPublishedDate = data.time[data['dist-tags'].latest];
+  const nxVersion = skipNxVersion || (await getNxVersion(data));
+  if (!data.repository) {
+    console.warn('- No repository defined in package.json!');
+    return { lastPublishedDate, nxVersion, githubRepo: '' };
   }
+  const url: String = data.repository.url;
+  const indexOfTree = url.indexOf('/tree/');
+  const githubRepo = url
+    .slice(0, indexOfTree === -1 ? undefined : indexOfTree)
+    .slice(0, url.indexOf('#') === -1 ? undefined : url.indexOf('#'))
+    .slice(url.indexOf('github.com/') + 11)
+    .replace('.git', '');
+  return {
+    lastPublishedDate,
+    githubRepo,
+    nxVersion,
+    // readmeContent: plugin.name
+  };
 }
 
 // Download count
 // i.e. https://api.npmjs.org/downloads/point/2023-06-01:2023-07-01/@nxkit/playwright
 async function getNpmDownloads(plugin: PluginRegistry) {
   const lastMonth = getLastMonth();
-  try {
-    const { data } = await axios.get(
-      `https://api.npmjs.org/downloads/point/${stringifyIntervalForUrl(
-        lastMonth
-      )}/${plugin.name}`
-    );
-    return data.downloads;
-  } catch (ex) {
-    return '';
-  }
+  const { data } = await axios.get(
+    `https://api.npmjs.org/downloads/point/${stringifyIntervalForUrl(
+      lastMonth
+    )}/${plugin.name}`
+  );
+  return data.downloads;
```

Comment:
We probably don't want to break our build if npm is down for what ever reason. We can use stale data.